src/lib/bits/aspect-ratio/components/aspect-ratio.svelte

```diff
@@ -12,7 +12,7 @@
 <div
 	style:position="relative"
 	style:width="100%"
-	style:padding-bottom="{100 / ratio}%"
+	style:padding-bottom="{ratio ? 100 / ratio : 0}%"
```

Comment:
Is there a reason why this component's implementation relies on the old `padding-bottom` trick instead of leveraging the [`aspect-ratio` property](https://caniuse.com/mdn-css_properties_aspect-ratio)?

Reply:
Apparently radix also does it like this. Maybe is for browser compatibility?
It is a trick from 2017 [CssTricks](https://css-tricks.com/aspect-ratio-boxes/)
But is widely accepted by all the browsers [CanIUse](https://caniuse.com/mdn-css_properties_aspect-ratio)

Reply:
Yeah maybe it's a good idea to use the aspect-ratio now that it has widespread support!